src/Realm/index.ts

```diff
@@ -1,12 +1,16 @@
-import { RealmResponse, CreateRealmPayload } from './types';
-import {
-  getRealm,
-  listRealms,
-  createRealm,
-  updateRealm,
-  deprecateRealm,
-} from './utils';
-import { CreateOrgPayload } from '../Organization/types';
+import { RealmResponse, CreateRealmPayload, RealmUtils } from './types';
+import store from '../store';
+import Store from '../utils/Store';
```

Comment:
We ought to find a better way to distinguish them based on more than the capitalization of the first letter...

Reply:
yup